pkg/quarterdeck/errors.go

```diff
@@ -0,0 +1,14 @@
+package quarterdeck
+
+// String errors intended to pass back from the server up to
+// a **human** Beacon user so these need to make sense to our customers!
+var (
+	ErrTryLoginAgain      = "Unable to login with those details - please try again!"
+	ErrTryProjectAgain    = "Unable to create or access that project - please try again!"
+	ErrNeedPermission     = "Unable to create or access projects - please request permission from your team owner."
```

Comment:
Should this message use `organization` instead of `team`?

Reply:
@daniellemaxwell we should use whichever term is used in the UI (i.e. if the UI calls them "Teams", this error message should reference "Teams"). So I think the real problem is that the UI references them as "Teams" in some places and "Organizations" in others. I've [created a story to address this](https://app.shortcut.com/rotational/story/16474/choose-either-teams-or-organizations-and-update-beacon-copy).